playbooks/group_vars/all/apt/external_repos.yaml

```diff
@@ -32,27 +32,27 @@ _apt_grafana_key:
     Cp9+ZHbR9yU+bkK0Dts4PNx4Wr9CktHIvbypT4Lk2oJEPWjcCJQHqpPQZXbnclXRlK5Ea0NVpaQd
     GK+vJS4HGxFFjSkvTKAZYgwOk93qlpFeDML3TuSgWxuw4NIDitvewudnaWzfl9tDIoVSBb16nwJ8
     bMDzovC/RBE14rRKYtMLmBsRzGYHWd0NnX+FitAS9uURHuFxghv9GFPheTaXvc4glM94HBUAEQEA
-    AbQmR3JhZmFuYSBMYWJzIDxlbmdpbmVlcmluZ0BncmFmYW5hLmNvbT6JAdQEEwEKAD4WIQS1Oud7
-    rbYwpoMEYAWWP6J3EEWFRQUCZOeGaQIbAwUJA8JnAAULCQgHAgYVCgkICwIEFgIDAQIeAQIXgAAK
-    CRCWP6J3EEWFRUiADACai+xytv2keEFJWjXNnFAx6/obnHRcXOI3w6nH/zL8gNI7YN5jcdQT2NYv
-    KVYTb3fWGuMsjHWgat5Gq3AtJrOKABpZ6qeYNPk0Axn/dKtOTwXjZ4pKX3bbUYvVfs0fCEZvB0HH
-    Ij2wI9kgMpoTrkj22LE8layZTPOoQ+3/FbLzS8hN3CYZj25mHN7bpZq8EbV38FW9EU0HM0tg6Cvo
-    xkRiVqAuAC0KnVIZAdhD4dlYKuncq64nMvT1A5wxSYbnE+ufmnWQQhhS6BOwRqN054yw1FrWNDFs
-    vnOSHmr8dIiriv+aZYvx5JQFJ7oZP3LwdYygocQcAJA8HFTIk3P6uJiIF/zdDzocgdKs+IYDoId0
-    hxX7sGCvqdrsveq8n3m7uQiN7FvSiV0eXIdV4F7340kc8EKiYwpuYSaZX0UWKLenzlUvD+W4pZCW
-    toXzPsW7PKUtq1xdW0+NY+AGLCvSJCc5F4S5kFCObfBAYBbldjwwJFocdq/YOvvWYTPyV7kJeJS5
+    AbQmR3JhZmFuYSBMYWJzIDxlbmdpbmVlcmluZ0BncmFmYW5hLmNvbT6JAdQEEwEKAD4CGwMFCwkI
+    BwIGFQoJCAsCBBYCAwECHgECF4AWIQS1Oud7rbYwpoMEYAWWP6J3EEWFRQUCaKhvPQUJB4NP1AAK
+    CRCWP6J3EEWFRUjOC/9YdWOWJLJVKzLx8uv5YVzebyw15HevhKahbznJX5fHnE8irjkiPFltVEZ4
+    T37s5afRGBEJnR1UFd80s7jzwbuoZh/zEB3jN8q50g64AznuzDa0PWKzaY7Tgkssx3+hs6TSvIwV
+    4z8T7f56lDudeHxHXx+htRnZ3ebKNPCJS7+G12GF6W3C3znpdjgvhVUB0uxd+42V0fRqk2GLNZeK
+    S9988fi5dYRAy9Ozwced7ByCFjde9FBgUtrH3mG1/ibzLEh04k02nYjc8mrH32t4UCWpxQEJ1vZA
+    2vT2HN3/cH/4uyFdyU6OHkMyMbz6lmeXe71dF5hOB4+/RP6Ndyj7ViRNDbm70NRBaFne/+YOJvmM
+    fJTCh7YbF5qEn1ihGkJJ0ohEu2IB+EGEhyiDm8SIsj1uMw7n17iIPNtbsU5GgnmLtfguP/WbwKV2
+    UeuxTpiOeYb6blDwRlh48uHMlA5HBW+487Jktw3iPj1IKhdtAC9CU3xAvzDcseMbgmM6Xj2bSQG5
```

Comment:
Is this supposed to be the contents of https://apt.grafana.com/gpg.key, or something else? (Asking because it doesn't seem to match.)

A description in the commit would be useful perhaps, to as what precisely is being done?